.eslintrc.yml

```diff
@@ -19,6 +19,7 @@ rules:
   valid-jsdoc: off
   no-multiple-empty-lines: off
   comma-dangle: off
+  no-unused-vars: off
```

Comment:
why do you added this ignore?

Reply:
There are *many* functions where we do things like:

```javascript
renderer: function(value, row) {
    // stuff here
    // we use "row" but not "value"
}
```

Reply:
Try prefixing the variable name with `_` this is an convention for intentionally unused variables like in this case.
```js
renderer: function(_value, row) {
    // stuff here
    // we use "row" but not "value"
}
```

Reply:
That's fair, but as we are moving away from raw javascript and refactoring the entire javascript codebase, the "reward to effort" ratio for fixing this is too low :)